modules/mongodb/templates/mongodb-backup-s3.erb

```diff
@@ -17,65 +17,109 @@ date_format = '%Y-%m-%d:%H:%M:%S'
 mongo_server = '<%= @server %>'
 key_fingerprint = '<%= @private_gpg_key_fingerprint %>'
 s3_bucket = '<%= @s3_bucket %>'
-standalone = '<%= @standalone %>'
+<% if standalone -%>
```

Comment:
Sorry, I still don't agree. I would argue that passing a config file is a better and more common practice (think init.d and upstart) than passing a bunch of command options. But still it's a matter of style which is subjective. For simplicity, I have opted to combine the config file and the code into the same file. The code has not changed, only the configuration variables.  

If anything, I would separate the config variables into a separate file and pass that as the only parameter. But why bother for a single purpose script when we've not done it elsewhere in the module?